superset/views/database/mixins.py

```diff
@@ -196,6 +204,9 @@ def _pre_add_update(self, database):
             check_sqlalchemy_uri(database.sqlalchemy_uri)
         self.check_extra(database)
         self.check_encrypted_extra(database)
+        database.server_cert = (
+            database.server_cert.strip() if database.server_cert else ""
+        )
```

Comment:
Good idea, I'll break it out into a separate util function.

Reply:
"server_cert" is not being validated before persisting to the DB, are we relying on `testconn`. It would be nice to add `parse_ssl_cert` to the pre_add, pre_update hooks